submissions/t-jay-777/Hooli-style Popup/style.less

```diff
@@ -0,0 +1,195 @@
+@width: 100vw;
+
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+body {
+	color: black;
+	font: 'Arial', sans-serif;
+}
+.container {
+	width: @width;
+	margin: 0 auto;
+}
+
+.nav {
+	.nav-menu {
+		display: flex;
+		flex-wrap: wrap;
+		flex-direction: row;
+		justify-content: flex-end;
+		margin: 15px;
+		li {
+			display: flex;
+			justify-content: center;
+			list-style-type: none;
+			margin: 1em;
+			a img:hover {
+				outline: 1px sold #00CED1;
+			}
+		}
+		a {
+			text-decoration: none;
+			color: #000000;
+		}
+		a:hover {
+			text-decoration-line: underline;
+		}
+		a img {
+			height: 1.5em;
+		}
+		.nav-submenu a img:hover {
+			outline: 1px solid #00CED1;
+		}
+	}
+	
+}
+
+.popup-menu-1::before {
+	content: '';
+    width: 1em;
+    height: 1em;
+    border-top: 1px solid #969599c2;
+    border-right: 1px solid #969599c2;
+	border: solid transparent;
+    position: absolute;
+    top: 9%;
+	/*
+	top: -8%
+    left: 50%;*/
+    background-color: #fff;
+	z-index: 99;
+	transform: rotate(-45deg);
+}
+.popup-menu-1 {
+	overflow-y: overlay;
+}
+
+.popup-menu {
+	max-width: @width/3;
+	height: 500px;
+	position: absolute;
+	top: 10%;
+	right: 15px;
+	background-color: #fff;
+	transition: opacity 300ms;
```

Comment:
Good practice to add an easing function, it's made your transitions more live and realistic.